api/app/controllers/v1/services_controller.rb

```diff
@@ -94,20 +93,7 @@ def sanitize_service_params(permitted_params)
       permitted_params[:user_id] = current_user.id unless current_user.admin?
       permitted_params.except!(:confirmation_date) unless current_user.admin?
 
-      associate_service_specification(permitted_params)
-    end
-
-    def associate_service_specification(service_params)
```

Comment:
I thought that the idea was that we do not expose the db id to frontend as the identification number is already a unique index